src/shared/src/language/texts/nn.ts

```diff
@@ -131,6 +131,7 @@ export function nn() {
       required: 'Obligatorisk',
       save: 'Lagre',
       save_and_close: 'Lagre og lukk',
+      save_and_next: 'Lagre og åpne neste',
```

Comment:
Maybe `Lagre og opne neste` instead? Tried to add a suggestion, but it seems to break:

```suggestion
      save_and_next: 'Lagre og opne neste',
```

Reply:
Ikke så god i nynorsk lengre virker det som